mlflow/models/model.py

```diff
@@ -551,6 +551,13 @@ def load(cls, path):
             # Load the Model object from a remote model directory
             model2 = Model.load("s3://mybucket/path/to/my/model")
         """
+
+        if not str(path).startswith(("s3://", "runs://")) and not os.path.exists(path):
+            raise MlflowException(
+                f'Could not find an "{MLMODEL_FILE_NAME}" configuration file at "{path}"',
+                RESOURCE_DOES_NOT_EXIST,
+            )
```

Comment:
It should support other URL protocols.

I think you don't need to validation path here, `download_artifacts` can validate it and raise proper error.

Reply:
without the path validation many tests fail. I'm happy to add a list of them. 

the challenge here is that I wanted to combine this common pattern of checking if the path exists with the actual loading of the file. it simplifies a lot of the code in the mlflow codebase this way. 

the other piece here is that the `os.path.exists` is only for local files, so i could use `"://" not in str(path)` to cover all of them

Reply:
Mlflow supports many kinds of artifact repo, adding full list here might not be a good idea.

> without the path validation many tests fail.

Why do they fail ? Current mlflow master CI is green.

Or you can check if it is a local path (i.e. extracting the URL scheme and check it is None or file), then checking `os.path.exists(path)` .

Reply:
ok, checking url scheme. please let me know if you have a better method you'd recommend.